config/docusaurus.config.js.rej

```diff
@@ -0,0 +1,19 @@
+diff a/config/docusaurus.config.js b/config/docusaurus.config.js	(rejected hunks)
```

Comment:
You seem to have unresolved merge conflicts in your source branch. Fix those first then this .rej file should disappear. Same for all other .rej files below.

Reply:
I see no conflics at all...

Reply:
Probably git resolved them automatically, but for some reason your editor couldn't delete them. Of you added them by mistake. I'd say it's safe to delete those .rej files since you already updated the applicable files - that's what matters